apps/codeforafrica/src/components/ConnectWithMember/ConnectWithMember.js

```diff
@@ -0,0 +1,34 @@
+import { RichTypography } from "@commons-ui/core";
+import Stack from "@mui/material/Stack";
+import React from "react";
+
+import SocialMediaBar from "@/codeforafrica/components/SocialMediaBar";
+import SocialMediaConnectButton from "@/codeforafrica/components/SocialMediaConnectButton";
+
+const ConnectWithMember = React.forwardRef(function ConnectWithMember(
+  props,
+  ref
+) {
+  const { sx, title, links, ...other } = props;
+  return (
+    <Stack
+      direction="row"
+      {...other}
+      sx={{
+        color: "grey.main",
+        alignItems: "center",
+        ...sx,
+      }}
+      ref={ref}
+    >
+      <RichTypography variant="footerCap">{title}</RichTypography>
+      <SocialMediaBar links={links}>
+        {Object.entries(links).map(([key, value]) => {
+          return <SocialMediaConnectButton key={key} name={key} url={value} />;
+        })}
+      </SocialMediaBar>
+    </Stack>
```

Comment:
```js
    <Stack
      direction="row"
      {...other}
      sx={{
        color: "grey.main",
        alignItems: "center",
        ...sx,
      }}
      ref={ref}
    >
      <RichTypography variant="footerCap">{title}</RichTypography>
      {children}
    </Stack>
```
This code is exactly the same here in `ConnectBar` and `ShareBar` (or `ShareThisPage` if I remember correctly). Extract it into a standalone component, `SocialMediaBar` and reuse it in these two components.